### DISCUSSION TITLE
contact email

### DISCUSSION THREAD

**Discussion by raynepaws** [cytrinc/bark-coding#49] [Show and tell]
i have created an email address that forwards all its received emails to bark mods. however, currently it only forwards to me because i don't know any other ppl's emails

email address is bark.github.io@gmail.com

**Comment by rubiidev18alt** [comment]
just go onto the profiles of people on the bark forums and add those emails to foward there

**Comment by raynepaws** [comment]
> just go onto the profiles of people on the bark forums and add those emails to foward there

but who is sodium

**Comment by mariocraft987** [comment]
mariocraft987dev@gmail.com
